source/development/advanced/known-issues.rst

```diff
@@ -45,8 +45,63 @@ Pushing attributes within other attributes `read_attribute` methods may cause de
 
 - **known workaround**: Push events for commands or own `read_attribute` method only.
 
+Using Network alias for TANGO_HOST and effect of FQDN names
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This issues affect to Tango systems whenever a network alias is used for the
+Tango Database together with fully qualified names (host.domain.suff).
+
+T7 server with no domain and client with domain trigger memory leaks in notifd
+
+In Tango 9 and server with domain, clients MUST use the FQDN; in opposite case you may loose events.
+
+Cells in red means likely Event_Timeout issues:
+
+.. image:: tango_alias_and_fqdn.png
+  :width: 600
+  :alt: Network alias and fqdn hostnames
+  
+Passing ZMQ events through firewalls, specific rules
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In ZMQ it is not possible to pass Tango events through an specific port, but
+specific firewall rules can be setup if each device server uses a different IP.
+
+- **known workaround**: it is possible to setup each device on a machine to use
+an specific IP address (to run servers on different IP addresses he server host 
+must have multiple interfaces, tipically virtual ones).
```

Comment:
It seem the identation has to be corrected (/home/travis/build/tango-controls/tango-doc/source/development/advanced/known-issues.rst:71: WARNING: Bullet list ends without a blank line; unexpected unindent.)
```suggestion
- **known workaround**: it is possible to setup each device on a machine to use
  an specific IP address (to run servers on different IP addresses he server host  
  must have multiple interfaces, tipically virtual ones).
```

Reply:
hi piotr, I agree with the change
thanks

Reply:
@sergirubio, I think it is you who should apply the suggestion as I do not have write permission to ALBA repo.